make/test.mk

```diff
@@ -303,8 +303,10 @@ create-host-resources:
 		echo "{\"spec\":{\"members\":{\"specificPerMemberCluster\":{\"member-$${TOOLCHAIN_CLUSTER_NAME}2\":{\"webhook\":{\"deploy\":false}}}}}}" > $$PATCH_FILE; \
 		oc patch toolchainconfig config -n $(HOST_NS) --type=merge --patch "$$(cat $$PATCH_FILE)"; \
 	fi;
-	# delete registration-service pods in case they already exist
+ifneq ($(E2E_TEST_EXECUTION),true)
+	# if it's not part of e2e test execution, then delete registration-service pods in case they already exist
```

Comment:
```suggestion
	# if it's not part of e2e test execution, then delete registration-service pods in case they already exist so that the ToolchainConfig will be reloaded
```